### ISSUE TITLE
How to scarpe metrics in mutliworker mode

### ISSUE THREAD

**Issue by bobykus31** [fluent/fluent-plugin-prometheus#90]
We run fluentd with 32 workers as it is a bit loaded, does it mean I need to have 32 HTTP endpoints configured on Prometheus server starting from  24231 when 

<source>
  @type prometheus
  bind 0.0.0.0
  port 24231
  metrics_path /metrics
</source>

Or is there  any way to accumulate metrics on one port?

**Comment by repeatedly** [comment]
> does it mean I need to have 32 HTTP endpoints configured on Prometheus server starting from 24231 when

Yes. See README: https://github.com/fluent/fluent-plugin-prometheus#prometheus-input-plugin

**Comment by xidiandb** [comment]
This means that /metrics only takes port 24231，If I use multiple workers, the url is useless

**Comment by ganmacs** [comment]
`aggregated_metrics_path` which was implemented so that users can get metrics from one port and one endpoint might help. https://github.com/fluent/fluent-plugin-prometheus/tree/31ca6065cee88a58d3d5a2dcebe095983b7cefda#prometheus-input-plugin

**Comment by Watson1978** [comment]
Hi, 

As mentioned by the maintainer above, please refer to the README for details on how to handle metrics with multiple workers. 

Since the question seems to be answered, I will go ahead and close this issue. If you have any further questions or run into issues, please feel free to reopen it or create a new one.

Thanks!
